pages/create-post.tsx

```diff
@@ -0,0 +1,43 @@
+import CreatePostWizard from "@components/create-post-page/CreatePostWizard";
+import Layout from "@components/Layout";
+import LoginModal from "@components/LoginModal";
+import pageTitle from "@lib/helpers/frontend/page-title";
+import { useUserQuery } from "@lib/hooks/user";
+import type { NextPage } from "next";
+import Head from "next/head";
+
+const PAGE_TITLE = "Create Post";
+
+const CreatePosting: NextPage = () => {
+  const { user } = useUserQuery();
+
+  if (!user) {
+    return (
+      <>
+        <Head>
+          <title>{PAGE_TITLE}</title>
+        </Head>
+        <Layout>
+          <LoginModal
+            isOpen={true}
+            message="To create a post you must be signed in. Please login with your MacID below."
+          />
+        </Layout>
+      </>
+    );
+  }
+
+  return (
+    <>
+      <Head>
+        <title>{pageTitle(PAGE_TITLE)}</title>
+        <link rel="icon" href="/favicon.ico" />
```

Comment:
Do we need to duplicate the favicon link tag here?

Reply:
Did realize that these where inherited